src/profile/vcpr.jl

```diff
@@ -0,0 +1,74 @@
+
+function profilevc(m::LinearMixedModel{T}, val::T, rowj::AbstractVector{T}) where {T}
```

Comment:
```suggestion
"""
     profilevc(m::LinearMixedModel{T}, val::T, rowj::AbstractVector{T}) where {T}

Profile an element of the variance components.

!!! note
   This method is called by `profile` and currently considered internal.
   As such, it may change or disappear in a future release without being considered breaking.
"""
function profilevc(m::LinearMixedModel{T}, val::T, rowj::AbstractVector{T}) where {T}
```